### PR TITLE
added unit tests to components

### PR DESCRIPTION
**Pull request by CHUKWUKA-EMI**
@rowlandekemezie, @chukwualukachiama

### CODE REVIEW

**Comment by CHUKWUKA-EMI** [comment]
@chialukachiama

File: src/setupTests.js [hunk 1]
```diff
@@ -2,4 +2,8 @@
 // allows you to do things like:
 // expect(element).toHaveTextContent(/react/i)
 // learn more: https://github.com/testing-library/jest-dom
-import '@testing-library/jest-dom/extend-expect';
+import "@testing-library/jest-dom/extend-expect";
+import { configure } from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
```
Comment (rowlandekemezie) [inline src/setupTests.js]: Remove this. only use @testing-library/react

File: src/Components/Trips/trips.test.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import { shallow, mount } from "enzyme";
+import { cleanup } from "@testing-library/react";
+
+import Trips from "./Trips";
+import Spinner from "../spinner/spinner";
+import Modal from "../Modal/modal";
+
+afterEach(cleanup);
+
+xtest("should display a header", () => {
```
Comment (rowlandekemezie) [inline src/Components/Trips/trips.test.js]: This test wouldn't run right

File: src/Components/Modal/Modal-Items/ModalItems.test.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+afterEach(cleanup);
+beforeEach(() => jest.setTimeout(90000));
+
+xtest("check items' properties", async () => {
+  const wrapper = shallow(<ModalItems />);
+
+  await expect(wrapper.find("button").getElement().props.children).toBe("Book");
+  await expect(wrapper.find("button").getElement().props.className).toBe("btn");
+  await expect(wrapper.find("input").getElement().props.type).toBe("checkbox");
```
Comment (rowlandekemezie) [inline src/Components/Modal/Modal-Items/ModalItems.test.js]: Why do you need await?

File: package.json [hunk 4]
```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@types/sinon": "^9.0.4",
```
Comment (rowlandekemezie) [inline package.json]: shouldn't it be in devDependencies. You don't need it tho'. Just nuke sinon completely

**Review by rowlandekemezie — COMMENTED** [review]
- [ ] Take time to read the documentation of @testing-library and follow the guide. 
- [ ] Try to review your PR on github. It should be clear that the test weren't all running because you added 'x(test|it)' to most of the test cases.
- [ ] Add your test script to also run on CircleCI

File: src/setupTests.js (hunk 1 above)
Comment (CHUKWUKA-EMI) [reply src/setupTests.js]: Alright sir

**Comment by CHUKWUKA-EMI** [comment]
> * [ ]  Take time to read the documentation of @testing-library and follow the guide.
> * [ ]  Try to review your PR on github. It should be clear that the test weren't all running because you added 'x(test|it)' to most of the test cases.
> * [ ]  Add your test script to also run on CircleCI

I actually added xtest whenever I was done with each test. I forgot to remove the 'x' before pushing to github.

File: src/Components/Modal/Modal-Items/ModalItems.test.js (hunk 3 above)
Comment (CHUKWUKA-EMI) [reply src/Components/Modal/Modal-Items/ModalItems.test.js]: It was telling me that jest async-await operation was timed-out. so when I set the timeout and added async-await to the test, it worked

File: src/Components/Trips/trips.test.js (hunk 2 above)
Comment (CHUKWUKA-EMI) [reply src/Components/Trips/trips.test.js]: It ran. But i added it after running the test, so as not to keep running it. I should have removed it before pushing to github.

File: package.json (hunk 4 above)
Comment (CHUKWUKA-EMI) [reply package.json]: Alright sir

**Comment by rowlandekemezie** [comment]
Was this close a mistake @CHUKWUKA-EMI ?

**Comment by CHUKWUKA-EMI** [comment]
@rowlandekemezie sir I'm very sorry for this. I didn't know it was closed. I didn't close it. I only renamed the branch so that circleCI could detect it. I think it got closed automatically.

**Pull request closed without merging**